pandas/formats/format.py

```diff
@@ -1,4 +1,8 @@
 # -*- coding: utf-8 -*-
+"""
+Internal module for formatting output data (e.g. `to_csv`).
+"""
```

Comment:
you can remove the ``to_csv`` (or add that it provides formatters for csv, html, latex and well as display formatting). It doesn't support ``to_csv``, rather the implemenation for it.